src/lib/letterAtPosition.ts

```diff
@@ -27,6 +30,22 @@ export enum LetterAtPositionInWord {
     Impossible = 'Impossible'
 }
 
+/**
+ * A comparator for sorting two `LetterAtPositionInWordRule` objects, following the usual
+ * pattern of positive (if the first rule is greater), positive (if the second rule is
```

Comment:
Incorrect doc: The comparator should/does return negative if the second one is greater.
```suggestion
 * pattern of positive (if the first rule is greater), negative (if the second rule is
```